pages/learn/develop/integrations/aws.md

```diff
@@ -46,16 +53,16 @@ Now it’s time to create some policies to allow our devices to communicate with
 
 For this project create a policy called `balena_control_policy`, and add the statement as shown below:
 
-| Action       | iot:* |
-|--------------|-------|
-| Resource ARN | *     |
-| Effect       | Allow |
+| Action       | iot:\* |
+| ------------ | ------ |
+| Resource ARN | \*     |
+| Effect       | Allow  |
 
 ![create policy](/img/integrations/aws/07_add_policy.png)
 
-Go to **Secure -> Certificates*. Select the recently created certificate and attach both the thing and policy to it.
+Go to \*_Secure -> Certificates_. Select the recently created certificate and attach both the thing and policy to it.
```

Comment:
Not sure this change was intentional and results in a leading * - maybe some auto-formatting by the editor?

Reply:
yeah, I will fix. thanks @garethtdavies